assets/scss/base/form/_hint.scss

```diff
@@ -126,3 +127,8 @@ Styleguide Form Hint.list
     }
   }
 }
+
+.form-hint-list-item--indented {
+  margin-left: em(30);
+  text-indent: em(-30);
+}
```

Comment:
This feels a little hacky.
One approach would be a mini `grid/table` with a small update to markup - being mindful of other services using the `.form-hint-list`


```
<ul class="form-hint-list">
  <li class="form-hint-list-item">
    <span class="form-hint-list-item__primary">
       <span class="form-hint-list-item__indicator"></span>
    </span>
     An example form hint, providing extra information to a user
  </li>
  <li class="form-hint-list-item {{modifier_class}}">
    <span class="form-hint-list-item__indicator"></span>
    A second example form hint, providing extra information to a user
  </li>
</ul>
```

then you can add to:
```
.form-hint-list-item {
   display: table;
}
```

and create: 
```
.form-hint-list-item__primary {
   display: table-cell;
}
```

This then will give you a 2 column layout.
you can then also potentially drop `vertical-align: middle;` from `.form-hint-list-item__indicator`